rimage/depth_map_test.go

```diff
@@ -187,13 +186,12 @@ func makeImagesForSubImageTest(ori, crop image.Rectangle) (*Image, *Image) {
 	return &Image{data: oriData, width: oriWidth, height: oriHeight}, &Image{data: cropData, width: cropWidth, height: cropHeight}
 }
 
-//nolint:dupl
 func makeDepthMapsForSubImageTest(ori, crop image.Rectangle) (*DepthMap, *DepthMap) {
 	oriWidth, oriHeight := ori.Max.X-ori.Min.X, ori.Max.Y-ori.Min.Y
 	overlap := ori.Intersect(crop)
 	cropWidth, cropHeight := overlap.Max.X-overlap.Min.X, overlap.Max.Y-overlap.Min.Y
-	oriData := make([]Depth, 0, oriWidth*oriHeight)
-	cropData := make([]Depth, 0, cropWidth*cropHeight)
+	oriData := []Depth{}
+	cropData := []Depth{}
```

Comment:
Nit:
```
	var oriData []Depth
	var cropData []Depth
```

Reply:
these will make oriData and cropData nil valued, and I need to compare them to empty slices (rather than nil slices)

Reply:
Ah, I see.